### ISSUE TITLE
Wrong path to Wine runner in .desktop files corresponding to Start Menu shortcuts

### ISSUE THREAD

**Issue by rdaysky** [lutris/lutris#2453]
1. Install an app that creates a Start Menu shortcut (so just about any app) using a Lutris script that involves a specific Wine runner
2. Observe that a .desktop file in ~/.local/share/applications/wine has been created
3. The .desktop file refers to the runner by a nonexistent path along the lines of:
```
[Desktop Entry]
Name=Some App
Exec=env WINEPREFIX="/home/user/lutris/someapp" /home/ubuntu/buildbot/runners/wine/tkg-unity-4.5-x86_64/bin/wine C:\\\\windows\\\\command\\\\start.exe ...
```
4. Instead, the file should have referred to the actual path of the runner, something under ~/.local/share/lutris/runners most likely.

**Comment by tannisroot** [comment]
The fact that those shortcuts get created at all is not the intended behavior.
I'm going to close this after I open an issue a proposal that should solve this.

**Comment by tannisroot** [comment]
Actually, now that I think of it, this is actually a pretty valid issue.
Still, I've opened an issue that's very related to this one: https://github.com/lutris/lutris/issues/2466

**Comment by strycore** [comment]
@tannisroot can this be closed? #2466 is fixed and I don't think there's anyway we could have our builds create valid wine shortcuts so it's best to just avoid them.

**Comment by tannisroot** [comment]
Yeah, closing.
